module-1/lab-functional-programming/your-code/main_bag_of_words_miriam.ipynb

```diff
@@ -0,0 +1,274 @@
+{
```

Comment:
1. You should include relative paths otherwise you get the error. It should look like the following. Try on your notebook.
```Python
docs = ['./doc1.txt', './doc2.txt', './doc3.txt']
```
2. Well done in your main bag_of_words. Congrats!!